### PR TITLE
Proposed vaccine author list

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Here is my proposed plan for author order. Please feel free to indicate if you think something doesn't look right. Here's the data I pulled:

Author/username | contributions | Order - Trad | Order - Novel | # lines
-- | -- | -- | -- | --
Anthony Gitter   | software and editing | 8 | 8 | 39
Casey Greene | mRNA | 7 | 9 |  
HM Rando   |   | 1 | 1 | 5514
Ronan Lordan  |   | 2 | 2 | 1055
likhithakolla   | mRNA | 5 | 3 | 116
Elizabeth (Ellie) Sell  | DNA | 4 | 4 | 42
Alexandra Lee  | whole-virus | 3 | 5 | 375
nilswellhausen    | novavax | 3 | 5 | 205
NAIKA86   | COVAXIN | 3 | 6 | 36
John J. Dziak   |   | ack | ack | 2
Consortium |   | 6 | 7 |  

I made a note where people had taken a major role in contributing to a particular vaccine write-up.

### Related issues

#830

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by RLordan — APPROVED** [review]

File: content/metadata.yaml [hunk 1]
```diff
@@ -261,6 +287,14 @@ authors:
         contributions:
           - Writing - Review & Editing
           - Visualization
+      vaccines-trad:
+        order: 3
+        contributions:
+          - Project Administration
```
Comment (agitter) [inline content/metadata.yaml:293]: Did Nils contribute to the writing for either manuscript? I see some lines edited in your summary table.

File: content/metadata.yaml [hunk 2]
```diff
@@ -419,6 +466,15 @@ authors:
     funders:
       - John W. and Jeanne M. Rowe Center for Research in Virology
     manuscripts:
+      vaccines-trad:
+        order: 8
```
Comment (agitter) [inline content/metadata.yaml:470]:
```suggestion
        order: 7
```

**Review by agitter — CHANGES_REQUESTED** [review]
We can continue the convention of Casey being last author.

File: content/metadata.yaml (hunk 1 above)
Comment (rando2) [inline content/metadata.yaml:293]:
```suggestion
          - Project Administration
          - Writing - Review & Editing          
```
Thank you! I missed this one!

**Review by agitter — APPROVED** [review]

**Pull request merged**